uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences-manager-leave-calendar.js

```diff
@@ -1,27 +1,13 @@
 var page = require('./page');
 
-var currentMonth = (new Date()).getMonth() + 1;
-
 module.exports = (function () {
   return page.extend({
     /**
      * Wait for the page to be ready
      */
     waitForReady: function () {
-      this.waitUntilVisible('.panel.panel-default:nth-of-type(' + currentMonth + ') .chr_leave-calendar__month-body');
-    },
-    /**
-     * Clear the months selected to show all months
-     */
-    showAllMonths: function () {
-      var casper = this.casper;
-      // If Current month is december then need to wait for November as it would be loaded last when all months load
-      var lastMonth = currentMonth === 12 ? 11 : 12;
-
-      casper.then(function () {
-        casper.click('.panel-subheading .close.ui-select-match-close');
-        casper.waitUntilVisible('.panel.panel-default:nth-of-type(' + lastMonth + ') .chr_leave-calendar__month-body');
-      });
+      // The end result of the page load is an item (cell) in the calendar
```

Comment:
can this be moved to the function comment? Something like:

`Wait for the page to be ready by watching the visibility leave calendar item element `
(this is a suggestion)